From 4a862609d3dc3eaba942f1379cbfde73485e40d6 Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Fri, 19 Aug 2016 14:15:15 +0100
Subject: [PATCH 1/1] MDL-12345 amos: good commit

This should be correct AMOS syntax

AMOS BEGIN
  CPY [searchengine,admin],[type_search,plugin]
  CPY [emailpasswordchangeinfosubject,core],[emailpasswordchangeinfosubject,auth_oauth2]
AMOS END
---
 lang/en/admin.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lang/en/admin.php b/lang/en/admin.php
index 1101ecf..2255a8c 100644
--- a/lang/en/admin.php
+++ b/lang/en/admin.php
@@ -935,7 +935,6 @@ $string['rssglobaldisabled'] = 'Disabled at server level';
 $string['save'] = 'Save';
 $string['savechanges'] = 'Save changes';
 $string['search'] = 'Search';
-$string['searchengine'] = 'Search engine';
 $string['searchinsettings'] = 'Search in settings';
 $string['searchresults'] = 'Search results';
 $string['searchsetupinfo'] = 'Search setup';
-- 
2.9.0

